packages/invoice-template/README.md

```diff
@@ -0,0 +1,281 @@
+# invoice-template
+
+Invoice templates created with react.
+
+## InvoiceBase
+
+Basic template component creates a document with logo, dates, seller and buyer info, and summary.
+
+```
+type InvoiceBaseProps = {
+    localizationOptions: LocalizationOptions;
+    invoiceValues: InvoiceValues;
+    invoiceItemsTable: TableDataProp;
+    taxesTable?: TableDataProp;
+};
+
+
+type LocalizationOptions = {
+    locale: string;
+    currency: string;
+    dateFormat: string;
+    documentDateLabel: string;
+    saleDateLabel: string;
+    dueDateLabel: string;
+    paymentMethodLabel: string;
+    sellerLabel: string;
+    buyerLabel: string;
+    totalLabel: string;
+};
+
+type InvoiceValues = {
+    logo?: ReactNode;
+    invoiceName: ReactNode;
+    documentDate: Date;
+    saleDate: Date;
+    dueDate: Date;
+    paymentMethod: ReactNode;
+    seller: ReactNode;
+    buyer: ReactNode;
+    total: ReactNode;
+};
+
+type TableDataProp = {
+    columns: TableColumns;
+    data: TableData;
+};
+
+type TableColumns = {
+    title: string;
+    dataIndex: string;
+    alignment?: "left" | "center" | "right";
+    width?: string;
+}[];
+
+type TableData = Record<string, string>[];
+```
+
+##### InvoiceBaseProps
+
+The type of component's props.
+
+##### LocalizationOptions
+
+Info about the
+
+-   locale (eg. `en_EN`)
+-   currency
```

Comment:
Here we also need to provide some more context. It isn't clear wheter this shoud be `$` or `USD` or something else.